docs/en_US/Release.md

```diff
@@ -1,5 +1,31 @@
 # ChangeLog
 
+## Release 0.9 - 7/1/2019
+
+### Major Features
+* General NAS programming interface
+    * Add `enas-mode` and `oneshot-mode` for NAS interface: [PR #1201](https://github.com/microsoft/nni/pull/1201#issue-291094510)
```

Comment:
> Add `enas-mode` 

Add `enas-mode` （TensorFlow only)

Reply:
Has the corresponding document been updated? May we add the instruction link here as well?

Reply:
will be tracked in issue #1233